### DISCUSSION TITLE
Spellpoints not Overwriting Spellslots

### DISCUSSION THREAD

**Discussion by JDHabit** [misthero/dnd5e-spellpoints#87] [Q&A] [answered]
Hello everyone,

I updated to the newest Version 11. 
I updated the Module.
I dragged the new Item into the character sheet.
I deleted the Old resource.
Then I tried to Cast a spell but it just asked me for Spellslots.

I'm confused about what I am doing wrong.
 
I do have the new Spellpoint Bar in the Charactersheet.
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/6a046bcb-9638-446c-b21a-3367a01e03bc)

I also got only the Module Enabled to test.
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/51d76c93-35d5-4703-92dc-cfaeab526aa4)

But when I cast the Spell I get this message:
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/cb875288-6193-4f9e-8b61-47c4dcb7d8c4)

I use up one of the spell slots like normal and the spell points don't reduce.
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/3083a424-1266-46ba-ae7e-ccf29e8c8472)

So what am I missing?

Help would be much appreciated ^-^"

**Comment by JDHabit** [comment]
Also in case it helps the current Foundry Version and Game System Version
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/af7821d1-f635-410a-8361-22f88f9be98c)

**Reply by misthero** [reply]
is the module active in his settings?

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/479ffd34-f240-4640-a534-3acdb34a2318)

**Reply by JDHabit** [reply]
Yes it is.
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/a9a0d756-0652-4c41-b94f-759be78cf0e2)

**Comment by misthero** [comment]
I think you need the new version of the module for dnd 3.2.0 but that version is also tied to foundry v12 so you won't see the update if you are on v11.

try this, download this version: https://github.com/misthero/dnd5e-spellpoints/releases/tag/v2.2.2

overwrite the version you have in data/modules/dnd5e-spellpoints/

edit the module.json file and change the minimum compatibile version from v12 to v11

from this:
```
 "compatibility": {
    "minimum": "12",
    "verified": "12.324"
  },
 ```
to this:
```
 "compatibility": {
    "minimum": "11",
    "verified": "12.324"
  },
 ```
let me know if that works

**Reply by JDHabit** [reply]
Took me a moment to find the time to do this. Sadly no change. Still the same as before.
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/6db54cb7-ed6e-4b83-afbe-a98ce9ce4752)

**Reply by misthero** [reply]
do you see any error in browser console when casting spells (F12 to open thge console).?

**Reply by JDHabit** [reply]
![image](https://github.com/misthero/dnd5e-spellpoints/assets/55274028/ba1b5803-8116-457d-b87a-ad6d6f6cc5a5)
I do get these 2 errors.

**Reply by misthero** [reply, marked as answer]
right, that's why the module is restricted to v12, the evaluate function to calculate the spell cost changed from v11 to v12, I was hoping it could work the same on 11 but it's not. so you cannot use this version of the module.

you either need to update foundry to v12, go back to a dnd version lower than 3.2 or I should make a special version that works on v11 and dnd3.2

**Reply by JDHabit** [reply]
Thanks for your time either way.

I guess I am gonna upgrade to V12 for now and see how that runs and if I can get all of my other Mods to function on V12 as well.

**Comment by misthero** [comment]
any news?
